src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -44,6 +44,10 @@ const customHTMLElementModels = {
         tagName: 'email-comment',
         mixedUAStyles: {whiteSpace: 'normal'},
     }),
+    strong: defaultHTMLElementModels.span.extend({
+        tagName: 'strong',
+        mixedUAStyles: {whiteSpace: 'pre'},
+    }),
```

Comment:
Are we good with this @alex-mechler ?

Reply:
I played with a variety of spacings with strong tags here and on main, and couldn't get this to break. I think this is fine. 
<img width="1122" alt="image" src="https://user-images.githubusercontent.com/22447860/226447616-0fb0c672-813c-4fba-a5d1-d5f2fa2ae104.png">
<img width="794" alt="image" src="https://user-images.githubusercontent.com/22447860/226447654-75d6f43f-f8b8-4a99-9310-4dce4d18512c.png">

Reply:
Cool, thanks!